client-app/shared/account/composables/useUser.ts

```diff
@@ -45,8 +49,42 @@ const organization = eagerComputed<Organization | null>(() => user.value?.contac
 const operator = computed<UserType | null>(() => user.value?.operator ?? null);
 
 export function useUser() {
+  const config = inject(configInjectionKey);
+
   const broadcast = useBroadcast();
   const { innerFetch } = useFetch();
+  const { openPopup, closePopup } = usePopup();
+
+  const changePasswordReminderDate: RemovableRef<Date | null> = useStorage("vcst-password-expire-reminder-date", null);
+
+  function handlePasswordExpiration(): void {
+    if (
+      user.value?.passwordExpiryInDays &&
+      user.value.passwordExpiryInDays <= (config?.password_expiry_in_days || 3) &&
```

Comment:
give a name to the constant "3"

Reply:
We don't need it at all, see comment below